GNUmakefile

```diff
@@ -20,3 +31,14 @@ testacc:
 .PHONY: updateclient
 updateclient:
 	go get github.com/yugabyte/platform-go-client
+
+#Build localy the provider
+.PHONY: build
+build:
+	go build -ldflags="-X 'main.version=v${VERSION}'" -o ${BINARY}
```

Comment:
Is this in line with the guidelines mentioned in the Contributing guideline (https://github.com/yugabyte/terraform-provider-yba/blob/main/CONTRIBUTING.md#building-locally). Would be helpful to keep this consistent.

Reply:
updated